### DISCUSSION TITLE
2022/01/29/mc%E5%BC%80%E6%9C%8D%E7%9A%84%E5%9F%BA%E6%9C%AC%E6%96%B9%E6%B3%95/

### DISCUSSION THREAD

**Discussion by giscus** [Westlifers/Westlifers.github.io#12] [Announcements]
# 2022/01/29/mc%E5%BC%80%E6%9C%8D%E7%9A%84%E5%9F%BA%E6%9C%AC%E6%96%B9%E6%B3%95/

在云服务器上安装Java环境下载JavaJAVA8：（Minecraft 1.16及以下请使用这个版本） JAVA16：（Minecraft 1.17及以上请使用这个版本） 如国外网站访问速度慢，可以在此网盘下载  注：1.18及以上使用Java18似乎报错，待查  安装Java 上传安装包到服务器/usr/local/java目录 使用以下命令解压该压缩包 1tar zxvf 文件名.tgz -

http://localhost:4000/2022/01/29/mc%E5%BC%80%E6%9C%8D%E7%9A%84%E5%9F%BA%E6%9C%AC%E6%96%B9%E6%B3%95/#screen%E4%BD%BF%E7%94%A8%E6%96%B9%E6%B3%95

**Comment by Westlifers** [comment]
深度好文！
